manifests/rhoai/params.env

```diff
@@ -1 +1 @@
-odh-training-operator-controller-image=quay.io/opendatahub/training-operator:v1-odh-c7d4e1b
+odh-training-operator-controller-image=quay.io/opendatahub/training-operator:v1.9.0-odh-1
```

Comment:
WDYT about using `quay.io/opendatahub/training-operator:v1-odh-750a731` to keep the same format?

Reply:
I assumed `v1.9.0-odh-1` is more human readable and makes it obvious if the image is up to date.

I don't have any strong opinion though so I'm OK to use the SHA tag if you prefer.

Reply:
I don't have any preference either, was just curious why the format changed